### PR TITLE
Updated GameDB.

### PR DESCRIPTION
**Pull request by YukiXXL**
Added fixes for Syphon Filter The Omega Strain Public Beta 1 and 2.

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -5563,6 +5563,19 @@ Serial = SCUS-97396
...
+Serial = SCUS-97397
+Name   = Syphon Filter - The Omega Strain Public Beta 1
+Region = NTSC-U
+EETimingHack = 1 // Fixes random hangs.
+[patches = C909A32E]
+	author=Refraction & YukiXXL
+	//COP2 fix.
+	patch=0,EE,00367798,word,48438000
+	patch=0,EE,0036779C,word,4B06521B
+	//Offline button fix
```
Comment (lightningterror) [inline bin/GameIndex.dbf:5575]: Maybe format the comments better so they match the rest of the file.

Comment (YukiXXL) [reply bin/GameIndex.dbf:5575]: Huh?

Comment (tlbjr) [reply bin/GameIndex.dbf:5575]: The one existing example I looked up had a space after `//` and before the actual comment.
```
// COP2 fix.
```

Comment (atomic83GitHub) [reply bin/GameIndex.dbf:5575]: Well, "COP2 fix" do not explain much what is going on here, I think you should write more about why does it need a patch, don't hesitate to contact the team as someone will probably know that.

**Comment by YukiXXL** [comment]
I don't know what I did wrong or what needs to be done... I'am lost O.o

**Comment by lightningterror** [comment]
+45,201 −45,180 changes, that doesn't look right.

**Review by lightningterror — CHANGES_REQUESTED** [review]
It's showing changes from other commits, I suggest you rebase with current master and then add your changes.

**Comment by lightningterror** [comment]
I'll close this since it's not being worked on, maybe someone else can add the patches instead.

**Pull request closed without merging**